data_model/src/query/mod.rs

```diff
@@ -421,7 +422,7 @@ impl_queries! {
     FindDomainById => crate::domain::Domain,
     FindDomainKeyValueByIdAndKey => MetadataValueBox,
     FindAllPeers => Vec<crate::peer::Peer>,
-    FindAllParameters => Vec<crate::parameter::Parameter>,
+    FindAllParameters => crate::param::Parameters,
```

Comment:
Maybe just `FindParameters`?

Reply:
other queries have `All` in them so I wouldn't change it. At least not in this PR